README.md

```diff
@@ -0,0 +1,169 @@
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+# My Budget App 🧮
+
+# 📗 Table of Contents
+  - [🛠️ Built With ](#-built-with-)
+  - [Tech Stack ](#tech-stack-)
+  - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [Video Presentation ](#video-presentation-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 My Budget App🧮 <a name="about-project"></a>
+
+Discover the power of effective financial management with **My Budget App**, a cutting-edge budgeting app designed to empower you on your journey towards financial wellness. With **My Budget App**, you're equipped with a comprehensive toolkit to take charge of your finances, make informed spending decisions, and achieve your financial goals.
+
+## 🛠️ Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client Side / Front-End</summary>
+  <ul>
+    <li><a href="https://https://https://gorails.com/">Ruby on Rails</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server Side / Back-End</summary>
+  <ul>
+    <li><a href="https://postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://render.com/">Render</a></li>
+    <li><a href="https://https://gorails.com/">Rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **Intuitive Budget Creation:** My Budget App makes budgeting a breeze. Create personalized budgets that align with your financial objectives using an easy and intuitive interface. Set spending limits for various categories and track your progress effortlessly.
+- [ ] **Real-Time Expense Tracking:** Keep tabs on your expenditures in real-time. Monitor where your money is going with automatic expense tracking that categorizes transactions and provides valuable insights into your spending habits.
+- [ ] **Customizable Categories:** Tailor your budget categories to match your lifestyle. My Budget App allows you to personalize spending categories to reflect your unique spending patterns and priorities
+- [ ] **User-Friendly Interface:** Navigating your finances has never been easier. My Budget App boasts an elegant and user-friendly interface that ensures a seamless experience for users of all levels.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://ror-budget-application.onrender.com)
+
```

Comment:
Please ensure the deployed version of your app is working as expected.
![Screenshot from 2023-08-19 21-47-14](https://github.com/Victoria129/budget-app/assets/44624138/fd9b60a9-64fd-477d-b2f8-80d25ff666c4)

Reply:
- [ ] As requested by the previous reviewer, kindly ensure the deployed version of your app is working as expected